example/Supfile

```diff
@@ -29,6 +29,8 @@ networks: # Groups of hosts
       - ubuntu@stg.example.com
 
   prod:
+    inventory:
+      - for i in 4 5 6; do echo "$USER@prod$i.example.com\n"; done
```

Comment:
the reasoning behind doing it as an array of commands was mostly since I felt it made the Supfile more readable. But :+1: for having it as a single command.